index.html

```diff
@@ -2072,20 +2072,18 @@ <h3>Extensibility</h3>
           <h4>Semantic Interoperability</h4>
 
           <p>
-This specification endeavors to enable both the JSON and JSON-LD syntaxes to
-be semantically compatible with one another without the JSON implementations
-needing to process the documents as JSON-LD. To achieve this, the
-specification imposes the following additional restrictions on both syntaxes:
+This specification ensures that "plain" JSON and JSON-LD syntaxes are semantically
+compatible without requiring JSON implementations to use a JSON-LD processor.
+To achieve this, the specification imposes the following additional restrictions,
+regarding the use of <code>@context</code>:
           </p>
 
           <ul>
             <li>
-JSON-based processors MUST process the <code>@context</code> <a>property</a>,
-ensuring the expected values exist in the expected order for the
-<a>credential</a> type being processed. It is advised that the expected order
-of values of the <code>@context</code> <a>property</a> should be defined by
-at least a human-readable extension specification and preferably by a
-machine-readable specification as well.
+JSON-based processors MUST process the <code>@context</code> <a>property</a>
+to the extent of ensuring the expected values exist in the expected order for
+the <a>credential</a> type being processed.
+              
```

Comment:
Why is the order of values important?

Reply:
JSON-LD typically works in a last-defined-wins model. There is a feature in JSON-LD 1.1, called `@protected` that works (more or less) in a first defined wins mode. So, order is important in both cases and if we enforce order, you don't need to use full blown JSON-LD processing if you just want to do a JSON-based processor.